features/step_definitions/cabinet_ministers_steps.rb

```diff
@@ -1,13 +1,27 @@
-Given(/^there are multiple Cabinet minister roles$/) do
-  organisation = create(:organisation)
-  create(:ministerial_role, name: "Role 1", cabinet_member: true, organisations: [organisation], seniority: 0)
-  create(:ministerial_role, name: "Role 2", cabinet_member: true, organisations: [organisation], seniority: 1)
+Given(/^reshuffle mode is (on|off)$/) do |on_or_off|
+  create(:sitewide_setting, key: :minister_reshuffle_mode, on: on_or_off == "on")
 end
 
 When(/^I visit the Cabinet ministers order page$/) do
   visit admin_cabinet_ministers_path
 end
 
+Then(/^I (should|should not) see a preview to the ministers index page$/) do |should_or_should_not|
+  selector = ".govuk-details__summary-text"
+  text = "Share ministers page preview"
+  if should_or_should_not == "should"
+    expect(page).to have_selector(selector, text:)
+  else
+    expect(page).to_not have_selector(selector, text:)
+  end
+end
```

Comment:
I feel this might actually be less, simpler, code as two separate methods rather than rolling `should`/`should not` into one

Reply:
Fair. I was trying to avoid having to duplicate the selector (less to update if we change the design) but it was a toss-up either way!